backend/src/main/java/it/chalmers/gamma/bootstrap/BootstrapConfig.java

```diff
@@ -0,0 +1,93 @@
+package it.chalmers.gamma.bootstrap;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component("cfg")
+public class BootstrapConfig {
```

Comment:
Also, are we sure this should all be bundled into one huge config class?

My thinking was more to split it into several:

AuthConfig => Settings regarding auth stuff goes here
OauthConfig => Oauth-stuff goes here
BootstrapConfig => Settings regarding mocking and bootstraping goes here

You sort of have a sane namespace in the config already, but it's not reflected in the code as is.

Reply:
Thing is that it all checks if there's an admin user, which is why I keep everything in one place. I think I will keep it like it is for now.